addons/cookoff/functions/fnc_detonateAmmunition.sqf

```diff
@@ -87,7 +87,7 @@ if (_amountOfMagazines > 0) exitWith {
         playSound3D [_sound, objNull, false, (getPosASL _vehicle), 2, 1, 1300];
 
         if (random 1 < 0.15) then {
-            [_vehicle, _ammo, _speed, random 1 < 0.15] call _spawnProjectile;
+            [_vehicle, _ammo, _speed, true] call _spawnProjectile;
```

Comment:
Don't need the _flyAway parameter in _spawnProjectile if we always set it to true.

Reply:
For `shotgrenade` and a few others it is still randomized. See L97ff